pkg/controller/defaults/resources.go

```diff
@@ -12,11 +12,11 @@ var (
 		"osd": corev1.ResourceRequirements{
 			Requests: corev1.ResourceList{
 				corev1.ResourceCPU:    resource.MustParse("1"),
-				corev1.ResourceMemory: resource.MustParse("4Gi"),
+				corev1.ResourceMemory: resource.MustParse("5Gi"),
 			},
 			Limits: corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("2"),
```

Comment:
The OSD will be Burstable Qos class in order to make it Guaranteed we should set the same value for the cpu as well.